### DISCUSSION TITLE
Nordigren API is configured incorrectly

### DISCUSSION THREAD

**Discussion by jaysbeekay** [firefly-iii/firefly-iii#8091] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Unable to import transactions from Nordigren having followed the instructions on the Firefly III website.  Looking through an extract of the logs, I can see it is looking for https://ob.nordigren.com however when looking into the API information for the Noridgren website (now Gocardless) it appears to have been changed to https://ob.gocardless.com.

### Debug information

`xxx.xxx.xxx.xxx - - [22/Oct/2023:16:06:50 +0100] "GET /authenticate HTTP/1.1" 200 2516 "http://xxx.xxx.xxx.xxx:8086/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.6 Safari/605.1.15"
127.0.0.1 - - [22/Oct/2023:16:07:03 +0100] "GET /health HTTP/1.1" 200 1105 "-" "Firefly III Health Checker/2.1"
[2023-10-22 16:07:26] local.DEBUG: Now in AuthenticateController, calling middleware.  
192.168.10.202 - - [22/Oct/2023:16:07:26 +0100] "POST /authenticate HTTP/1.1" 302 2509 "http://192.168.20.119:8086/authenticate" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.6 Safari/605.1.15"
[2023-10-22 16:07:27] local.DEBUG: Now in AuthenticateController, calling middleware.  
[2023-10-22 16:07:27] local.DEBUG: Now at App\Services\Nordigen\AuthenticationValidator::validate  
[2023-10-22 16:07:27] local.DEBUG: Now at App\Services\Nordigen\TokenManager::hasValidRefreshToken  
[2023-10-22 16:07:27] local.DEBUG: No Nordigen refresh token, so return false.  
[2023-10-22 16:07:27] local.DEBUG: No refresh token, so return false.  
[2023-10-22 16:07:27] local.DEBUG: Now at App\Services\Nordigen\TokenManager::getNewTokenSet  
[2023-10-22 16:07:37] local.ERROR: cURL error 6: Could not resolve host: ob.nordigen.com (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://ob.nordigen.com/api/v2/token/new/ {"exception":"[object] (GuzzleHttp\\Exception\\ConnectException(code: 0): cURL error 6: Could not resolve host: ob.nordigen.com (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://ob.nordigen.com/api/v2/token/new/ at /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:210)
[stacktrace]`

### Expected behaviour

Upon submitting Nordigren ID and Secret, I should be able to authenticate and receive correct transaction listing for my linked accounts

### Steps to reproduce

1. Insert Nordigren key and secret into Data Import tool
2. Authenticate
3. Error message appears - "Nordigren API is configured incorrectly"

**Comment by JC5** [comment]
> Could not resolve host: ob.nordigen.com

Since this URL still works fine for me, I expect this is a local internet hiccup.

**Reply by jaysbeekay** [reply]
Doing a DNS lookup on ob.nordigren.com fails. I have tried from both my local machine and mxtoolbox.com. 

<img width="1310" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/46616837/30204af2-1bab-4de8-a737-22dacbfd4bc7">


Looking into the documentation here https://developer.gocardless.com/bank-account-data/endpoints, the API endpoint has been changed to https://ob.gocardless.com which resolves correctly.

**Reply by JC5** [reply]
You misspelled the URL.

**Reply by jaysbeekay** [reply]
You are indeed correct - turned out to be a DNS issue with my container (and poor eyesight on my part!)

**Reply by mikedolx** [reply]
Hi,

i'm stuck with a similar issue. I upgraded my data-importer to the version 1.5.2. Now im am faced with the following error-message on my importer frontend:

![grafik](https://github.com/firefly-iii/firefly-iii/assets/15738117/6cde373f-0220-41be-ad21-f51e9b5e7b64)

Do i have to change the url, or is it hard-coded in the code?

I am running firefly and the data-importer in a docker environment. I have connected "nicolaka/netshoot" to the network and was able to ping the hostname `ob.gocardless.com`. I also restarted the container, but with no luck. Any idea?

**Reply by JC5** [reply]
Did you press the "start over" button?
